src/modules/local_position_estimator/BlockLocalPositionEstimator.cpp

```diff
@@ -426,29 +515,31 @@ void BlockLocalPositionEstimator::updateHome()
 	_altHome = alt;
 	_gpsAltHome += delta_alt;
 	_baroAltHome +=  delta_alt;
-	_lidarAltHome +=  delta_alt;
-	_sonarAltHome +=  delta_alt;
+	_visionHome(2) += delta_alt;
```

Comment:
This correction is basically an offset based on the height of the sensor above the ground. It really makes more sense to be a static parameter. If we try to auto calculate it when the sensor initializes. This leads to unreliable offsets while taking off for sonar and lidar if they are too close to the ground when the vehicle is landed to initialize.